packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -55,11 +64,44 @@ export interface _AttachmentCardProps {
 
 /**
  * @internal
- * A component for displaying a attachment card with attachment icon and progress bar.
+ * A component for displaying an attachment card with attachment icon and progress bar.
  */
 export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
-  const { attachmentName, attachmentExtension, progress, actionIcon } = props;
-  const theme = useTheme();
+  const { attachmentName, attachmentExtension, progress, actionIcon, actionHandler } = props;
+  // const { file, progress, menuActions, onDownloadErrorMessage } = props;
```

Comment:
Should we clear up these comments?

Reply:
These are to be used by John later on as part of the breaking API changes, but will clean this up